NugetLightProvider.cs

```diff
@@ -368,37 +368,59 @@ public void FindPackage(string name, string requiredVersion, string minimumVersi
 
             try {
 
-            
-            // If there are any packages, yield and return
-            if (request.YieldPackages(request.GetPackageById(name, request, requiredVersion, minimumVersion, maximumVersion, minInclusive, maxInclusive), name))
-            {
-                return;
-            }
+                // If there are any packages, yield and return
+                if (request.YieldPackages(request.GetPackageById(name, request, requiredVersion, minimumVersion, maximumVersion, minInclusive, maxInclusive), name))
+                {
+                    return;
+                }
 
-            // Check if the name contains wildcards. If not, return. This matches the behavior as "Get-module xje" 
-            if (!String.IsNullOrWhiteSpace(name) && !WildcardPattern.ContainsWildcardCharacters(name))
-            {              
-                return;
-            }
+                // If no packages were found, try again using credentials retrieved from credential provider
+                // First call to the credential provider is to get credentials, but if those credentials fail,
+                // we call the cred provider again to ask the user for new credentials, and then search pkgs again using new creds
+                var credentials = request.GetCredsFromCredProvider(request.Sources.First(), request, false);
```

Comment:
Make sure this code is specific to Azure Cred provider only. We do not impact  the existing find-package -credential scenario.